app/src/main/res/values/strings.xml

```diff
@@ -77,4 +77,7 @@
     <!-- TODO to jest tekst nie zatwierdzony przez nikogo -->
     <string name="thank_you_for_teaching">Dziękujemy!</string>
     <string name="informations">Informacje</string>
+    <string name="pola_application">Aplikacja Pola</string>
+    <string name="version">wersja</string>
+    <string name="kj_copyright">© Klub Jagielloński</string>
```

Comment:
Sorry za czepliwość, ale lepiej będzie jak wrzucisz to jako jeden string z parametrami, czyli:
<string name="pola_application">Aplikacja Pola\nwersja %1$s (%2$d)\n(c) Klub Jagielloński</string>
I wtedy użyć w kodzie jako getString(R.string.pola_aplication, versionName, versionCode);

Reply:
done